### PR TITLE
Update to Turing.jl GSOC projects

### PR DESCRIPTION
**Pull request by torfjelde**
As it says in the title!:)

Sorry for the delay; a lot of people to sync up with here.

### CODE REVIEW

**Review by logankilpatrick — APPROVED** [review]

File: jsoc/gsoc/turing.md [hunk 1]
```diff
@@ -1,112 +1,131 @@
...
+**Project length:** 175 hrs or 350 hrs
+
+**Description:**
+GPU support in Turing is not quite there yet for several reasons:
+
+Bijectors.jl, the package which provides transformations of distributions to Turing.jl, is not fully compatible with GPU. For example, many of the transformations make use of scalar indexing which is slow on GPU.
+DynamicPPL.jl, the package providing the DSL of Turing.jl, is not compatible with GPU. Again, a lot of scalar indexing is used, and likely some internal functions are simply just incompatible with GPU usage at the moment.
+Others?
+
+There might also be other issues along the way, and making Turing.jl fully support GPU usage within the span of the project is very unlikely, _but* taking a significant step in this direction should be possible and will be very useful.
```
Comment (logankilpatrick) [inline jsoc/gsoc/turing.md:72]:
```suggestion
There might also be other issues along the way, making Turing.jl fully support GPU usage within the span of the project is very unlikely, but taking a significant step in this direction should be possible and will be very useful.
```

File: jsoc/gsoc/turing.md [hunk 2]
```diff
@@ -1,112 +1,131 @@
...
-**Description**: 
-Although KernelFunctions.jl has extensive correctness testing, our performance testing is lacking. This project aims to resolve this, and resolve performance issues wherever they are found. The contributor would first need to extend our existing benchmarking coverage, and debug any obvious performance problems. The next phase of the work would be to construct end-to-end examples of KernelFunctions being used in practice, profile them to determine where performance problems lie, and fix them.
+**Project length:** 350 hrs
 
-## Iterative methods for inference in Gaussian Processes
+**Description:**
+At the moment there is currently no support for running a Turing model in a “batched” mode.
 
-**Mentors**: Will Tebbutt, S. T. John, Ross Viljoen
+When one wants to run, say, 2 chains in parallel for a given model, the current approach is to in effect to call sample(model, ...) twice.
```
Comment (logankilpatrick) [inline jsoc/gsoc/turing.md:100]:
```suggestion
When one wants to run, say, 2 chains in parallel for a given model, the current approach is to in effect to call `sample(model, ...)` twice.
```

File: jsoc/gsoc/turing.md [hunk 3]
```diff
@@ -1,112 +1,131 @@
...
-**Project difficulty**: Hard
+**Project difficulty:** Medium
 
-**Project length**: 350 hrs
+**Project length:** 350 hrs
 
-**Description**:
-Adding [approximate inference](https://github.com/JuliaGaussianProcesses/JuliaGaussianProcesses.github.io/discussions/5#discussioncomment-1627101) methods for non-Gaussian likelihoods which are available in other GP packages but not yet within JuliaGPs. The project would start by determining which approximate inference method(s) to implement---there's lots to do, and we're happy to work with a contributor on whichever method they are most interested in, or to suggest one if they have no strong preference.
+**Description:**
+Adding [approximate inference](https://github.com/JuliaGaussianProcesses/JuliaGaussianProcesses.github.io/discussions/5#discussioncomment-1627101) methods for non-Gaussian likelihoods which are available in other GP packages but not yet within JuliaGPs. The project would start by determining which approximate inference method(s) to implement---there’s lots to do, and we’re happy to work with a contributor on whichever method they are most interested in, or to suggest one if they have no strong preference.
```
Comment (logankilpatrick) [inline jsoc/gsoc/turing.md:120]:
```suggestion
Adding [approximate inference](https://github.com/JuliaGaussianProcesses/JuliaGaussianProcesses.github.io/discussions/5#discussioncomment-1627101) methods for non-Gaussian likelihoods which are available in other GP packages but not yet within JuliaGPs. The project would start by determining which approximate inference method(s) to implement - there’s lots to do, and we’re happy to work with a contributor on whichever method they are most interested in, or to suggest one if they have no strong preference.
```

**Comment by logankilpatrick** [comment]
This is failing CI for some reasons

**Comment by torfjelde** [comment]
Thanks for the typo-fixes!

Uhmm is there a way to debug this?

**Comment by logankilpatrick** [comment]
Thank you!

**Pull request merged**